hw/lint/data/common_lint_cfg.hjson

```diff
@@ -17,11 +13,11 @@
   build_dir:        "{scratch_path}/{build_mode}"
 
   // we rely on fusesoc to run lint for us
-  build_cmd:  "fusesoc "
+  build_cmd:  "{tool_srcs_dir}/run-in-bash.sh {build_dir}/lint-{tool}/{tool}-tool.log fusesoc "
```

Comment:
How about we add `build_log` variable in the Hjson and supply it to `lint.mk`? We could change [lint.mk](https://github.com/lowRISC/opentitan/blob/6b869134f355fe3e6e79d64a7a5e4b33480e3e7b/hw/lint/data/lint.mk#L22) to this:
```
	-cd ${build_dir} && ${build_cmd} ${build_opts} | tee ${build_log}
```

Reply:
good idea. I'll do that for synthesis as well (#1749), since DC does not support writing logfiles for the main shell except for a command log, which is of no use in this case.

Reply:
Ok, I have added a mechanism to override the default title postfix which is "Lint Results" for the lint class. For Verible, I now override this with "Style Lint Results". Also, if you click on one of the modules on the dashboard, the page that comes up with the detailed messages also indicates which tool was used (this would be "veriblelint" in this case).

@sriyerg I have added a field named `results_title_postfix` to the `OneShotCfg` class. If that field is empty, the default string of each subclass is used, e.g. "Lint Results"  or "Synthesis Results". Let me know if you think we should do this differently. We can also add this to the base class if this could be useful for other flows.